src/components/custom/_ctrader-wrapper.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { nav_style, content_style } from './_ctrader-wrapper.module.scss'
+import Logo from 'images/svg/ctrader/logo.svg'
+import Watermark from 'images/svg/ctrader/watermark.svg'
+import Flex from 'features/components/atoms/flex-box'
+
+const CtraderWrapper = ({ children }: { children: React.ReactNode }) => (
+    <>
+        <Flex.Box className={nav_style} justify="center" align="center" gap="5x">
+            <img src={Logo} alt="ctrader" width="43" height="43" />
```

Comment:
@mohammad-hashemi-deriv we have two image components `<Icon>` and other is `<Image>` can we use here any of them.